@jufrantz There are compiler errors (on Linux) with your changes:
```
.../maya-usd/lib/usd/ui/layerEditor/layerTreeItem.cpp: In member function ‘void UsdLayerEditor::LayerTreeItem::populateChildren(UsdLayerEditor::RecursionDetector*)’:
.../maya-usd/lib/usd/ui/layerEditor/layerTreeItem.cpp:155:25: error: possibly dangling reference to a temporary [-Werror=dangling-reference]
  155 |             const auto& stageRootLayerStack
      |                         ^~~~~~~~~~~~~~~~~~~
.../maya-usd/lib/usd/ui/layerEditor/layerTreeItem.cpp:156:85: note: the temporary was destroyed at the end of the full expression ‘(& pxrInternal_v0_25_11__pxrReserved__::UsdStage::GetPseudoRoot() const().pxrInternal_v0_25_11__pxrReserved__::UsdPrim::GetPrimIndex())->pxrInternal_v0_25_11__pxrReserved__::PcpPrimIndex::GetRootNode().pxrInternal_v0_25_11__pxrReserved__::PcpNodeRef::GetLayerStack()’
  156 |                 = _stage->GetPseudoRoot().GetPrimIndex().GetRootNode().GetLayerStack();
      |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
cc1plus: all warnings being treated as errors
```

--- a/lib/usd/ui/layerEditor/layerTreeItem.cpp
+++ b/lib/usd/ui/layerEditor/layerTreeItem.cpp
@@ -152,7 +152,7 @@ void LayerTreeItem::populateChildren(RecursionDetector* recursionDetector)
         // composed from root and session layer variables.
         std::string resolvedPath = path;
         if (_stage && SdfVariableExpression::IsExpression(path)) {
-            const auto& stageRootLayerStack
+            const auto stageRootLayerStack
                 = _stage->GetPseudoRoot().GetPrimIndex().GetRootNode().GetLayerStack();
 
             if (stageRootLayerStack) {

